operators/from_file.go

```diff
@@ -0,0 +1,43 @@
+// Copyright 2022 Juan Pablo Tosso and the OWASP Coraza contributors
+// SPDX-License-Identifier: Apache-2.0
+
+package operators
+
+import (
+	"os"
+	"path"
+)
+
+func loadFromFile(filepath string, paths []string) ([]byte, error) {
```

Comment:
Might consider accepting `[]fs.FS` instead of strings for paths, this would allow someone to use an `go:embed` file system as roots if they needed to.

Reply:
Not sure I understand the use case. This is more like an internal function that is used by the `*FromFile` functions, I don't think people might call that directly.